CacheManager.StackExchange.Redis.SelfCleaning/SelfCleaningRedisCacheHandle.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using CacheManager.Core;
+using CacheManager.Core.Internal;
+using CacheManager.Core.Logging;
+using CacheManager.Redis;
+using CacheManager.StackExchange.Redis.SelfCleaning.Core;
+using CacheManager.StackExchange.Redis.SelfCleaning.Timers;
+using StackExchange.Redis;
+
+namespace CacheManager.StackExchange.Redis.SelfCleaning
+{
+    public class SelfCleaningRedisCacheHandle<TCacheValue> : RedisCacheHandle<TCacheValue>, IStartable
```

Comment:
You can add summary that explain why that class even exists (the problem about Redis timeout mechanism), and add that this class assume that the Redis doesnt clean itself (the items being inserted without any expiration request)

Reply:
Added